infrastructure-tests/runTests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh -e
+
+# Sanity check
+if [ -z "$PPIPER_INFRA_IT_CF_USERNAME" ]; then
+    echo "Failure: Variable PPIPER_INFRA_IT_CF_USERNAME is unset"
+    exit 1
+fi
+
+if [ -z "$PPIPER_INFRA_IT_CF_PASSWORD" ]; then
+    echo "Failure: Variable PPIPER_INFRA_IT_CF_PASSWORD is unset"
+    exit 1
+fi
+
+set -x
+
+# Start a local registry, to which we push the images built in this test, and from which they will be consumed in the test
+docker run -d -p 5000:5000 --restart always --name registry registry:2 || true
+find ../cx-server-companion -type f -exec sed -i "" -e 's/ppiper/localhost:5000\/ppiper/g' {} \;
+
+# Configure our testing Jenkins instance via Configuration as Code (Create build job, register shared libraries, configure executors)
+mkdir -p ../cx-server-companion/life-cycle-scripts/jenkins-configuration
+cp testing-jenkins.yml ../cx-server-companion/life-cycle-scripts/jenkins-configuration
```

Comment:
Might, took me a few seconds to understand that you are copying it there to configure the cx-server

Reply:
I would rather copy the cx-server here.

Reply:
I don't see why this would be easier to understand.

Reply:
For example it will also put the jenkins configuration into the companion image. Not sure if this can lead to side effects. 
I think it is cleaner to copy them into some temporary folder as the other folder is part of a docker image.